enh(mc): refresh InfluxDB page (#2599)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-influxdb.md

id: applications-databases-influxdb
title: InfluxDB
---
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Contenu du pack

InfluxDB est une base de données de séries chronologiques (généralement abrégé en TSDB pour time series database) sous licence MIT.

> 

Reviewer: ```suggestion
```

| Connection-Time        | App-DB-Influxdb-Connection-Time-custom        | Contrôle le temps de connexion à l'instance    |            |
| Database-Statistics    | App-DB-Influxdb-Database-Statistics-custom    | Contrôle les statistiques des bases de données | X          |
| Http-Server-Statistics | App-DB-Influxdb-Http-Server-Statistics-custom | Contrôle les statistiques du serveur HTTP      |            |
| Write-Statistics       | App-DB-Influxdb-Write-Statistics-custom       | Contrôle les statistiques d'écriture           |            |

> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **App-DB-Influxdb-custom** est utilisé.

> Si la case **Découverte** est cochée, cela signifie qu'une règle de découverte de service existe pour ce service.

</TabItem>
<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">

| Alias | Modèle de service            | Description                                                                                             |
|:------|:-----------------------------|:--------------------------------------------------------------------------------------------------------|

> | Query | App-DB-Influxdb-Query-custom | Contrôle permettant d'exécuter des requêtes et d'utiliser le résultat pour définir des seuils d'alarmes |

Reviewer: ```suggestion
| Query | App-DB-Influxdb-Query-custom | Contrôle permettant d'exécuter des requêtes et d'utiliser le résultat pour définir des seuils d'alarme |
```

| *databases*#database.series.count       | count |

</TabItem>
<TabItem value="Http-Server-Statistics" label="Http-Server-Statistics">

| Métrique                        | Unité |
|:--------------------------------|:------|
| requests.query.count.persecond  | N/A   |
| requests.write.count.persecond  | N/A   |
| requests.ping.count.persecond   | N/A   |
| requests.status.count.persecond | N/A   |
| requests.active.count           | count |
| requests.write.active.count     | count |
| requests.response.data.bytes    | B/s   |
| requests.write.data.bytes       | B/s   |
| errors.server.persecond         | N/A   |
| errors.client.persecond         | N/A   |

</TabItem>
<TabItem value="Query" label="Query">

| Métrique                 | Unité |
|:-------------------------|:------|
| *queries_results*#status | N/A   |

</TabItem>
<TabItem value="Write-Statistics" label="Write-Statistics">

| Métrique                 | Unité |
|:-------------------------|:------|
| points.written.persecond | N/A   |
| writes.ok.persecond      | N/A   |
| writes.error.persecond   | N/A   |
| writes.drop.persecond    | N/A   |
| writes.timeout.persecond | N/A   |

</TabItem>
</Tabs>

## Prérequis

> *Specify prerequisites that are relevant. You may want to just provide a link\n\

Author: ```suggestion
Pour pouvoir accéder aux données d'une base InfluxDB, il faut généralement disposer d'un login/mot de passe ou d'un token.
```

Author: Is it enough?

> *Specify prerequisites that are relevant. You may want to just provide a link\n\

Reviewer: no text here

> to the manufacturer official documentation BUT you should try to be as complete\n\

Reviewer: ```suggestion
```

> as possible here as it will save time to everybody.*

Reviewer: ```suggestion
```

| WARNINGREQUESTSWRITEDATA     | Warning threshold                                                                                   |                   |             |
| CRITICALREQUESTSWRITEDATA    | Critical threshold                                                                                  |                   |             |
| EXTRAOPTIONS                 | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |

</TabItem>
<TabItem value="Query" label="Query">

| Macro          | Description                                                                                                                                                            | Valeur par défaut | Obligatoire |
|:---------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | INSTANCE       | Set the instance label on which the results should be calculate for (Example: --instance='name').  The instance label must be the same label as the "GROUP BY" keyword |                   | X           |

Reviewer: ```suggestion
| INSTANCE       | Set the instance label for which the results should be calculated (Example: --instance='name').  The instance label must be the same label as the "GROUP BY" keyword |                   | X           |
```

> | OUTPUT         | Set the output for each instances (Example: --output='Object %{instance} value is {label}')                                                                            |                   | X           |

Reviewer: ```suggestion
| OUTPUT         | Set the output for each instance (Example: --output='Object %{instance} value is {label}')                                                                            |                   | X           |
```
